src/game/camera.c

```diff
@@ -12337,6 +12345,22 @@ void mode_rom_hack_camera(struct Camera *c) {
     pos[1] = mPos[1] + desiredHeight;
     pos[2] = mPos[2] + sins(sRomHackYaw) * desiredDist;
 
+    // Move camera down for hangable ceilings
+    if (sMarioCamState->action & ACT_FLAG_HANGING) {
```

Comment:
not sure what happened here, but isn't this code already merged?

Reply:
Not sure either